LocalReferenceLinkRel/explainer.md

```diff
@@ -0,0 +1,105 @@
+# Local References In <link> Tags
+
+## Authors:
+
+- Andy Luhrs
+- Kurt Catti-Schmidt
+
+## Participate
+- [Issue tracker](https://github.com/MicrosoftEdge/MSEdgeExplainers/labels/LRLR)
+- [Discussion forum](https://github.com/whatwg/html/issues/11019)
+
+## Status of this Document
+
+This document is intended as a starting point for engaging the community and
+standards bodies in developing collaborative solutions fit for standardization.
+As the solutions to problems described in this document progress along the
+standards-track, we will retain this document as an archive and use this section
+to keep the community up-to-date with the most current standards venue and
+content location of future work and discussions.
+
+* This document status: **Active**
+* Expected venue: [WHAT Working Group](https://whatwg.org/)
+* Current version: this document
+
+## Introduction
+Modern web development practices have converged towards building reusable components instead of building monolithic documents. Frameworks such as React and Angular have popularized this approach, which has made its way natively to the web platform via Web Components. 
+
+Web Components allow for defining custom elements, and these custom elements often rely on Shadow DOM for ID encapsulation. But Shadow DOM doesn't just encapsulate ID's - it encapsulates style as well. The current preferred approach
+for styling elements under a Shadow DOM is through `<link>` tags, but these tags have several downsides, namely that they must be an external file or a dataURI. The performance and developer ergonomics of this
+situation are not ideal, and developers using Custom Elements have expressed strong interest in addressing these issues.
+
+This explainer proposes a solution to this situation by allowing another option for sharing styles via the `<link>` tag - local references to `<style>` objects.
+
+## Goals
+* Allow for developers to share styles defined in the Light DOM document into Shadow DOM trees.
+* Allow developers to define their CSS upfront in the same document as their HTML when using Shadow DOM.
+* Define a mechanism that will allow for more selective style sharing via CSS `@sheet`.
+
+## Non-goals
+* Anything specific to `@sheet` should be discussed in its dedicated proposal.
```

Comment:
```suggestion
* Anything specific to `@sheet` should be discussed in its dedicated <a href="https://github.com/MicrosoftEdge/MSEdgeExplainers/blob/main/AtSheet/explainer.md">proposal</a>.
```

Reply:
On second thought, maybe better would be to linkify `@sheet` in the last item under "Goals."

Reply:
I think it'd be fine to link in both places.